assets/styles/callouts.scss

```diff
@@ -0,0 +1,350 @@
+:root {
```

Comment:
this file doesn't seem to be imported in `partials/head.html`, can you add this to that partial and include a feature flag for it?

Reply:
Added